gammapy/utils/time.py

```diff
@@ -0,0 +1,55 @@
+# Licensed under a 3-clause BSD style license - see LICENSE.rst
+"""Time related utility functions."""
+from __future__ import (absolute_import, division, print_function,
+                        unicode_literals)
+from astropy.time import Time, TimeDelta
+
+__all__ = ['time_ref_from_dict',
+           'time_relative_to_ref',
+           ]
+
+
+def time_ref_from_dict(meta):
+    """Calculate the time reference from metadata.
+    The time reference is built as MJDREFI + MJDREFF in units of MJD.
+    All other times should be interpreted as seconds after the reference.
+
+    Parameters
+    ----------
+    meta : dict
+    	dictionary with the keywords `MJDREFI` and `MJDREFF`
+
+    Returns
+    -------
+    `~astropy.time.Time` object with the reference time in format of `MJD`.
```

Comment:
This doesn't look too bad in the html docs, see [here](https://dl.dropboxusercontent.com/u/4923986/temp/gammapy-278/html/api/gammapy.utils.time.time_ref_from_dict.html).

But I still think the normal pattern is to have the same format for "Returns" as for "Parameters", in this case:
```
time : `~astropy.time.Time`
    Reference time with ``format='MJD'``.
```
Or did you see this other format for "Returns" docstrings in many places in Numpy / Scipy / Astropy?